twilio/Module.md

```diff
@@ -27,38 +23,44 @@ If you use a trail account, you may need to verify your recipient phone numbers
 
 # Quickstart
 
-To use the Twilio connector in your Ballerina application, update the .bal file as follows:
+To use the Twilio connector in your Ballerina application, update the `.bal` file as follows:
 
-### Step 1 - Import connector
-Import the Twilio module to your Ballerina program as follows. You can use [configurable variables](https://ballerina.io/learn/by-example/configurable.html) to provide the necessary credentials.
+### Step 1 - Import the package
+
+Import the Twilio package to your Ballerina program as follows. You can use [configurable variables](https://ballerina.io/learn/by-example/configurable.html) to provide the necessary credentials.
 
 ```ballerina
 import ballerinax/twilio;
 ```
 
 ### Step 2 - Create a new connector instance
+
 To create a new connector instance, add a configuration as follows:
+
 ```ballerina
 configurable string accountSId = ?;
 configurable string authToken = ?;
 
 twilio:ConnectionConfig twilioConfig = {
     auth: {
-        accountSId: accountSId,
-        authToken: authToken
+        accountSId,
+        authToken
     }
 };
 
-twilio:Client twilioClient = new (twilioConfig);
+twilio:Client twilio = new (twilioConfig);
```

Comment:
Is this (not using the `Client` suffix in client endpoint variable naming) a convention we should be using in all the code samples?

Reply:
Yes, we are using it all new samples we are writing. For ex: refer https://github.com/ballerina-guides/integration-samples/blob/main/github-pull-requests-to-gsheets/main.bal 

So make sure to follow those in upcoming sample writing efforts as well.

Reply:
@anupama-pathirage Ack :) will make sure to follow the same convention in future efforts